model/src/commonMain/kotlin/io/github/droidkaigi/feeder/TimetableItem.kt

```diff
@@ -9,20 +9,62 @@ sealed class TimetableItem(
     open val title: MultiLangText,
     open val startsAt: Instant,
     open val endsAt: Instant,
+    open val category: TimetableCategory,
+    open val targetAudience: String,
+    open val language: String,
+    open val asset: TimetableAsset,
+    open val room: TimetableRoom,
+    open val levels: List<String>,
 ) {
     data class Session(
         override val id: String,
         override val title: MultiLangText,
         override val startsAt: Instant,
         override val endsAt: Instant,
+        override val category: TimetableCategory,
+        override val targetAudience: String,
+        override val language: String,
+        override val asset: TimetableAsset,
+        override val room: TimetableRoom,
+        override val levels: List<String>,
+        val description: String,
         val speakers: List<TimetableSpeaker>,
-    ) : TimetableItem(id, title, startsAt, endsAt)
+        val message: MultiLangText?,
+    ) : TimetableItem(
+        id = id,
+        title = title,
+        startsAt = startsAt,
+        endsAt = endsAt,
+        category = category,
+        targetAudience = targetAudience,
+        language = language,
+        asset = asset,
+        room = room,
+        levels = levels,
+    )
 
     data class Special(
         override val id: String,
         override val title: MultiLangText,
         override val startsAt: Instant,
         override val endsAt: Instant,
+        override val category: TimetableCategory,
+        override val targetAudience: String,
+        override val language: String,
+        override val asset: TimetableAsset,
+        override val room: TimetableRoom,
```

Comment:
There will be no parallel sessions at DroidKaigi 2021. Therefore, no room is needed. 🙇

Reply:
Oh, I see.
I deleted TimetableRoom class.